src/statsig/constants.ts

```diff
@@ -15,15 +15,24 @@ export const LayerParams = {
   },
 }
 
-export const ExperimentParams = {
+export const ExperimentConfigs = {
```

Comment:
is it possible to define a type for this that ensures the key and value of experimentName field are the same?